torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10220,19 +10343,23 @@ def ref_pairwise_distance(input1, input2):
            dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_masked_fill,
            supports_forward_ad=True,
+           supports_fwgrad_bwgrad=False,  # need: _s_where
            # https://github.com/pytorch/pytorch/issues/66357
            check_batched_forward_grad=False,
            supports_out=False),
     OpInfo('masked_scatter',
            dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_masked_scatter,
            supports_forward_ad=True,
+           supports_fwgrad_bwgrad=True,
            # https://github.com/pytorch/pytorch/issues/66357
            check_batched_forward_grad=False,
            supports_out=False),
     OpInfo('masked_select',
            dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            supports_forward_ad=True,
+           # RuntimeError: ZeroTensors are immutable. Please use the materialized zero tensor (...)
```

Comment:
Can you please open an issue for this so we can track them on the side?

Reply:
https://github.com/pytorch/pytorch/issues/69855